test_openapi3.yml

```diff
@@ -256,6 +256,9 @@ components:
     apiToken:
       type: http
       scheme: bearer
+    basic:
```

Comment:
Can we also apply this to a route, similar to how we did in the test_swagger.yml one?

Reply:
resolved in https://github.com/acr-lfr/generate-it/pull/150